codes/quantum/oscillators/fock_state/knill-laflamme-milburn-loqc.yml

```diff
@@ -0,0 +1,71 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+code_id: knill_laflamme_milburn_loqc
+physical: qubits
+logical: qubits
+
+name: 'Knill-Laflamme-Milburn Linear Optical QC encoding'
+introduced: '\cite{doi:10.1038/35051009}' 
+
+
+description: 'Qubits are encoded with dual-rail logic i.e. using the physical location of a photon in two spatial modes. Logical \(|\bar{0}\rangle\) is encoded as \(|01\rangle_{ab}\) and logical \(|\bar{1}\rangle\) is encoded as \(|10\rangle_{ab}\), where a and b are spatial modes. By performing non-linear gates adaptively, the Knill-Laflamme-Milburn protocol approximates linear gates for universal quantum computation. For scalability, the protocol is enhanced by using teleportation to increase success probability of the adaptive circuit.
+
+The KLM protocol is further enhanced by applying explicit error-correction after teleportation, by concatenating an outer stabilizer code with the dual-rail logic code. Logical \(|\bar{0}\rangle\) is encoded as \(\left|\Phi^{+}\right\rangle\), and logical \(|\bar{1}\rangle\) is encoded as \(\left|\Psi^{+}\right\rangle\).'
```

Comment:
Phi and Psi are undefined here

Reply:
thanks for pointing that out, I'm adding the clarification